libs/shared/feature/schema-nuv/src/lib/components/schema-select/formio/formio-schema-select.component.ts

```diff
@@ -25,28 +26,39 @@ export class FormioSchemaSelectComponent extends FormioBaseCustomComponent<ISche
   public formControl = new FormControl();
   public gotSchemas = false;
 
-  constructor(private readonly _workApiRoutesService: WorkApiRoutesService, private readonly _changeDetectorRef: ChangeDetectorRef) {
+  private schemaKey = '';
+  private readonly getSchemaKey$ = this._schemaBuilderService.metaDataFields$.pipe(tap(data => (this.schemaKey = data.key)));
+
+  constructor(
+    private readonly _workApiRoutesService: WorkApiRoutesService,
+    private readonly _changeDetectorRef: ChangeDetectorRef,
+    private readonly _schemaBuilderService: SchemaBuilderService,
+  ) {
     super();
   }
 
   public ngOnInit(): void {
+    this.getSchemaKey$.subscribe();
```

Comment:
Lets prevent the memory leak
```suggestion
    this.getSchemaKey$.pipe(take(1)).subscribe();
```